Formula/gds-cli.rb

```diff
@@ -8,8 +8,9 @@ class GdsCli < Formula
   head "git@github.com:alphagov/gds-cli.git", :using => :git
 
   depends_on "go" => :build
-  tap_name = OS.mac? ? "homebrew/cask" : "linuxbrew/extra"
-  depends_on "#{tap_name}/aws-vault"
+  if !OS.mac? then
```

Comment:
```suggestion
  unless OS.mac? then
```

Reply:
I'm not a huge `unless` fan but sure whatever.  I didn't mean to commit the `then` though.

Reply:
🤔 actually, even better would be `if OS.linux?` but I can't find the documentation for this `OS` class.  Where does it even come from?

Reply:
https://github.com/Homebrew/brew/blob/6b2dbbc96f7d8aa12f9b8c9c60107c9cc58befc4/Library/Homebrew/os.rb